src/mbed_tools/cli/build.py

```diff
@@ -111,6 +111,7 @@ def build(
     if flash:
         for dev in devices:
             flash_binary(dev.mount_points[0].resolve(), program.root, build_tree, mbed_target, hex_file)
+        click.echo(f"{len(devices)} file(s) copied ({'.HEX' if hex_file else '.BIN'}).")
```

Comment:
Can we not just display the full path(s) of the files copied? This log message looks like it will display e.g "1 file(s) copied (.HEX)", which is more useful than no message but isn't really that useful for debugging etc.